core/src/main/scala/org/apache/spark/util/collection/Spillable.scala

```diff
@@ -41,7 +41,7 @@ private[spark] abstract class Spillable[C](taskMemoryManager: TaskMemoryManager)
   protected def forceSpill(): Boolean
 
   // Number of elements read from input since last spill
-  @volatile protected def elementsRead: Long = _elementsRead
```

Comment:
It's not meaningful to make a method volatile anyway right? the fact that it's not an error makes me think I could be missing something. No objection